WordPressAuthenticator/OAuth/OAuthTokenResponseBody.swift

```diff
@@ -1,7 +1,7 @@
 /// Models the response to an OAuth token request.
 ///
 /// - Note: See documentation at https://developers.google.com/identity/protocols/oauth2/native-app#exchange-authorization-code
-struct OAuthTokenResponseBody: Decodable {
+struct OAuthTokenResponseBody: Codable, Equatable {
```

Comment:
I don't like that this is `Codable` and `Equatable` in the production code, when those capabilities are only needed in the tests. However, this is the only way to get the compiler to do the work of conforming to the protocols for us.